spec/spec_helper.rb

```diff
@@ -17,6 +17,10 @@
 # If you are not using ActiveRecord, you can remove this line.
 ActiveRecord::Migration.check_pending! if defined?(ActiveRecord::Migration)
 
+GovukAbTesting.configure do |config|
+  config.acceptance_test_framework = :active_support
+end
```

Comment:
this might make more sense in a `spec/support/govuk_ab_testing.rb` file rather than here.